.github/workflows/update-homebrew-formula-on-release.yml

```diff
@@ -0,0 +1,19 @@
+name: Update Homebrew Formula On Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  update-homebrew-formula-on-release:
+    runs-on: macos-latest
+    steps:
+      - name: Update Homebrew formula
+        uses: dawidd6/action-homebrew-bump-formula@v3
```

Comment:
Was this image suggested from the homebrew folks? I'd rather not use something that a user can freely modify, can the homebrew folks publish one instead?

Reply:
This was suggested by @SMillerDev who is a member of Homebrew. But I agree with you!

@SMillerDev can you help us?

https://github.com/Homebrew/homebrew-core/pull/73610#issuecomment-803846210

Reply:
@Dawidd6 is a homebrew maintainer too. But you can always tag it to a known good commit.

Reply:
Yep, @SMillerDev is right. Feel free to @ me if you run into any issues.

Reply:
Thanks guys! I think we can insert a tag in order to be more specific.

WDYF @vincepri?

Reply:
Hi @SmillerDev @dawidd6 (thanks for all your hard work on homebrew first off!), I want to check with folks in Kubernetes ContribX to make sure we're ok using third party code actions. Have you folks considered hosting the action under the @Homebrew organization?

Reply:
That would still be a third party :smile:. Dawidd made the action and it isn't an official Homebrew action (those are only written for Homebrew CI). If he wants we can probably move it but there isn't anything to gain from it being in the homebrew organisation as far as I can tell.

Reply:
If you are really concerned about security, then just pin the Action to specific commit as Sean said earlier. Moving the Action to Homebrew org was something I considered in the past but nah, no specific gain from it.